napalm_logs/config/iosxr/INTERFACE_DOWN.yml

```diff
@@ -0,0 +1,13 @@
+messages:
+  # 'error' should be unique and vendor agnostic. Currently we are using the JUNOS syslog message name as the canonical name.
+  # This may change if we are able to find a more well defined naming system.
+  - error: INTERFACE_DOWN
```

Comment:
You should be able to combine both `up` and `down` into one error. Catch the state as a variable and cast it to `|upper` like https://github.com/napalm-automation/napalm-logs/blob/master/napalm_logs/config/junos/INTERFACE_DOWN.yml#L10

Reply:
Actually, on second thoughts this would mean that we would have a different error name for interface down on junos and iosxr. What do you think @mirceaulinic?